dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -0,0 +1,31 @@
+from io import open, StringIO, BytesIO
+
+from . import data_utils
+
+class FileOrBufferHandler():
+    def __init__(self, stream_or_buffer, open_method='r', options=dict()):
+        self._stream_or_buffer = stream_or_buffer
+        self.open_method = open_method
+        self.options = options
+
+    def __enter__(self):
+        if (data_utils.is_stream_buffer(self._stream_or_buffer)):
+            print(self.options)
+            if 'seek' in self.options:
+                self._stream_or_buffer.seek(self.options['seek'],0)
```

Comment:
utilize the above suggestion for the input args and make sure there's a space after the comma